### DISCUSSION TITLE
unable to clone in wsl2

### DISCUSSION THREAD

**Discussion by ceimos** [community/community#70294] [New to GitHub]
So, when I execute the following command in windows cmd

git clone "repo-address"
It executes perfectly, does not even ask me for username and password. When I do the same in WSL2 (ubuntu) I get the following error -

Cloning into 'LearnGit'...
/mnt/c/Program\ Files/Git/mingw64/bin/git-credential-manager-core.exe get: 1: /mnt/c/Program Files/Git/mingw64/bin/git-credential-manager-core.exe: not found
Username for 'https://github.com': ceimos
Password for 'https://ceimos@github.com':
/mnt/c/Program\ Files/Git/mingw64/bin/git-credential-manager-core.exe erase: 1: /mnt/c/Program Files/Git/mingw64/bin/git-credential-manager-core.exe: not found
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/ceimos/LearnGit.git/'
exit status 128
git-credential-manager-core.exe did not exist at the mentioned directory so i did this, after reading few stackoverflow posts -

git config --global credential.helper "/mnt/c/Program\ Files/Git/mingw64/libexec/git-core/git-credential-wincred.exe
but now I get this error -

Cloning into 'LearnGit'...
usage: git credential-wincred <get|store|erase>

/mnt/c/Program\ Files/Git/mingw64/libexec/git-core/git-credential-wincred.exe
 get: 2: get: not found
Username for 'https://github.com': ceimos
Password for 'https://ceimos@github.com':
usage: git credential-wincred <get|store|erase>

/mnt/c/Program\ Files/Git/mingw64/libexec/git-core/git-credential-wincred.exe
 erase: 2: erase: not found
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/ceimos/LearnGit.git/'

I know it says " support for password authentication was removed". but, i am worried because it works on windows.

**Comment by thevindu-w** [comment]
When it asks for password, enter a PAT (Personal Access Token) instead of the account password.
You can create a token from: [https://github.com/settings/tokens](https://github.com/settings/tokens)
